pkg/broker/util.go

```diff
@@ -43,6 +43,82 @@ func toBrokerPlans(apbPlans []apb.Plan) []Plan {
 	return brokerPlans
 }
 
+func updateMetadata(metadata map[string]interface{}, params []apb.ParameterDescriptor) map[string]interface{} {
+	if params == nil || len(params) == 0 {
+		return metadata
+	}
+
+	if metadata == nil {
+		metadata = make(map[string]interface{})
+	}
+
+	formDefinition := make([]interface{}, 0)
+
+	for paramIdx := 0; paramIdx < len(params); {
+		var formItem interface{}
+		pd := params[paramIdx]
+		if pd.DisplayGroup == "" {
+			formItem, paramIdx = createUIFormItem(pd, paramIdx)
```

Comment:
The reason I'm modifying the paramIdx, is that the two functions consume a differing amount of items from the list.  If it was a formItem, it consumed 1 item.  If it was a group, it was an arbitrary number of items that matched.